### PR TITLE
frida-python: modify setup.py for offline egg installation

### PR DESCRIPTION
**Pull request by 0xABD**
dirty hack for offline install: search for the egg file in the current folder

### CODE REVIEW

**Comment by oleavr** [comment]
Thanks! I like this solution, let's go for it.

File: src/setup.py [hunk 1]
```diff
@@ -61,8 +61,14 @@ def build_extension(self, ext):
                 os_version=os_version
             )
 
-            print('downloading prebuilt extension from', egg_url)
-            egg_data = urlopen(egg_url).read()
+            try:
+                print('downloading prebuilt extension from', egg_url)
+                egg_data = urlopen(egg_url).read()
+            except:
+                egg_url = re.sub(r".*/f/frida/(.*)","file:///\g<1>",egg_url)
```
Comment (oleavr) [inline src/setup.py]: Nitpick: missing space after commas for coding style consistency.

**Comment by 0xABD** [comment]
Thank you, corrected the style.

**Comment by oleavr** [comment]
Thanks! :+1:

**Pull request merged**